src/components/image-config/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+import ImageConfig from '.'
+
+const testImage = (
+  <img alt="hashicorp logo" src="/img/favicons/favicon-192x192.png" />
+)
+
+/**
+ * this is an approximation of this component's markup when used within an MDX document, eg:
+ * <ImageConfig>
+ *
+ *   ![hashicorp logo](/img/favicons/favicon-192x192.png)
+ *
+ * </ImageConfig>
+ */
+const renderBasicMock = () =>
+  render(
+    <ImageConfig hideBorder>
+      <p className="g-type-long-body">{testImage}</p>
+    </ImageConfig>
+  )
+
+describe('basic functionality', () => {
+  //  silence console.warn
+  const mockConsoleWarn = jest.spyOn(console, 'warn')
+  beforeAll(() => mockConsoleWarn.mockImplementation(() => null))
+  afterAll(() => mockConsoleWarn.mockRestore())
+
+  it('should render children', () => {
+    render(<ImageConfig>{testImage}</ImageConfig>)
+
+    expect(screen.getByAltText('hashicorp logo')).toBeInTheDocument()
+  })
+
+  it('should apply `noBorder` styling if `hideBorder` is provided', () => {
```

Comment:
`hideBorder` is a deprecated property for this image config in Learn. I'm not sure the history there but I see so many warnings about this in the build logs. I wonder if we should just do away with it altogether in dev portal.

Reply:
Perhaps we need to carry it over though since its still used in the content